client/landing/stepper/declarative-flow/internals/steps-repository/choose-a-plan/index.tsx

```diff
@@ -1,308 +1,26 @@
-			].filter( isValueTruthy );
-		};
-
-		return (
-			<div className="plans-grid">
-				<PlansIntervalToggle
-					intervalType={ billingPeriod }
-					onChange={ onPlansToggleChanged }
-					maxMonthlyDiscountPercentage={ maxAnnualDiscount }
-					className="plans-grid__toggle"
-				/>
-
-				<div className="plans-grid__table">
-					<div className="plans-grid__table-container">
-						<div className="plans-table">
-							{ filteredPlans
-								.filter( ( plan ) => !! plan )
-								.map( ( plan, index ) => (
-									<span
-										key={ 'plan-item-' + plan.periodAgnosticSlug }
-										id={ 'plan-item-' + plan.periodAgnosticSlug }
-									>
-										<PlanItem
-											popularBadgeVariation="ON_TOP"
-											allPlansExpanded={ allPlansExpanded }
-											key={ plan.periodAgnosticSlug }
-											slug={ plan.periodAgnosticSlug }
-											domain={ domain }
-											CTAVariation="NORMAL"
-											features={ getVideoPressFeaturesList( plan ) }
-											billingPeriod={ billingPeriod }
-											isPopular={ 'business' === plan.periodAgnosticSlug }
-											isFree={ plan.isFree }
-											name={ plan?.title.toString() }
-											isSelected={
-												!! selectedPlanProductId &&
-												selectedPlanProductId ===
-													getPlanProduct( plan.periodAgnosticSlug, billingPeriod )?.productId
-											}
-											onSelect={ ( id ) => onPlanSelect( id, plan ) }
-											onPickDomainClick={ () => goToStep && goToStep( 'chooseADomain' ) }
-											onToggleExpandAll={ () => setAllPlansExpanded( ( expand ) => ! expand ) }
-											// translators: Placeholder refers to the name of a WordPress.com plan.
-											CTAButtonLabel={ __( 'Get %s' ).replace( '%s', plan.title ) }
-											popularBadgeText={ __( 'Best for Video' ) }
-										/>
-										{ index < filteredPlans.length - 1 && (
-											<div key={ 'plan-item-separator-' + index } className="plan-separator"></div>
-										) }
-									</span>
-								) ) }
-						</div>
-					</div>
-				</div>
-			</div>
-		);
-	};
-
-	const getFormattedHeader = () => {
-		if ( ! isVideoPressFlow ) {
-			return undefined;
-		}
-
-		return (
-			<FormattedHeader
-				id="choose-a-plan-header"
-				headerText={ __( 'Choose a plan' ) }
-				subHeaderText={ __( 'Unlock a powerful bundle of features for your video site.' ) }
-				align="center"
-			/>
-		);
-	};
-
-	const stepContent = isVideoPressFlow ? getVideoPressFlowStepContent() : getDefaultStepContent();
+const ChooseAPlan: Step = function ChooseAPlan( { navigation } ) {
+	const { goNext, goBack } = navigation;
 
 	return (
 		<StepContainer
 			stepName="chooseAPlan"
-			shouldHideNavButtons={ isVideoPressFlow }
+			shouldHideNavButtons={ false }
 			goBack={ goBack }
 			goNext={ goNext }
 			isHorizontalLayout={ false }
 			isWideLayout={ true }
 			isLargeSkipLayout={ false }
-			stepContent={ stepContent }
+			stepContent={ <h1>Choose a plan step</h1> }
```

Comment:
Can this be translated?

Reply:
I moved the thing from a const to here, but I can surely add a translation (I'll add a note for another PR so it doesn't delay this one), I'm wondering if this step is even used tbh